lib/src/container/store.rs

```diff
@@ -441,8 +441,13 @@ impl ImageImporter {
         imgref: &OstreeImageReference,
         mut config: ImageProxyConfig,
     ) -> Result<Self> {
-        // Apply our defaults to the proxy config
-        merge_default_container_proxy_opts(&mut config)?;
+        if imgref.imgref.transport == Transport::ContainerStorage {
+            //Fetching from containers-storage, may require priviliges to read files
```

Comment:
Still need space between comment and text here.

Reply:
ah yes! :)